### ISSUE TITLE
How do I run Presentation Mode?

### ISSUE THREAD

**Issue by ivorpad** [FormidableLabs/spectacle#436]
Unfortunately the instructions aren't clear enough and the presentation doesn't sync.

What I'm doing:

1. Open two browser windows in two different screens (MacBook Pro and Dell P2715Q through mDP)
2. Visit http://localhost:3000 then add `?presenter` immediately after `/#/`
3. Open http://localhost:3000 in the second browser window (also known as the presentation screen).

I expect both presentations to be in sync, but they are not. Am I missing something? I also tried via `ngrok` but I get the same results.

Thanks a lot for the great work!

**Comment by sw-yx** [comment]
are you running this in a server? it depends on socket.io to sync the two things.

**Comment by ivorpad** [comment]
@sw-yx Yes, I'm running on a server and also tried with `ngrok`

**Comment by TomiS** [comment]
I think there might be some things broken because I'm getting this error
```
manager.js:335 Uncaught TypeError: Cannot read property 'forward' of null
    at Manager._goToSlide (manager.js:335)
```
on viewer tab console when I click right arrow on presenter tab.

**Comment by sw-yx** [comment]
hmm you're right. i thought it worked when i tried it out.  now it no longer works.

**Comment by TigerWolf** [comment]
I fixed this issue by updating to `4.0.4`

**Comment by sneakers-the-rat** [comment]
Replying here (no need to reopen) because it's one of the first results for "spectacle.js presenter mode" to say the url is now

`/?presenterMode=true`

https://formidable.com/open-source/spectacle/docs/basic-concepts/#presenting
